src/python/pants/core/goals/lint.py

```diff
@@ -335,13 +311,7 @@ class LintSubsystem(GoalSubsystem):
 
     @classmethod
     def activated(cls, union_membership: UnionMembership) -> bool:
-        return bool(
-            {
-                LintRequest,
-                FmtTargetsRequest,
-                _FmtBuildFilesRequest,
-            }.intersection(union_membership.union_rules.keys())
-        )
```

Comment:
Why is this safe?

Reply:
It isn't (but also it is, lol) I'll just put it back for now